content/blog/2021/04/2021-04-12-jenkins-operator-sub-project.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Jenkins Operator becoming sub-project!"
+tags:
+- kubernetes
+- operator
+- contributing
+authors:
+- antoniaklja
+- sylwiabrant
+sig: cloud-native
+opengraph:
+  image: /images/logos/kubernetes-operator/kubernetes-operator.png
```

Comment:
It won't look well either 16x9 is the best resolution

Reply:
done, I've added 16:9 resolution